build.rs

```diff
@@ -6,7 +6,7 @@ fn main() {
     // Add the toolchain lib/ directory to `-L`.  This fixes the linker error "cannot find
     // -lLLVM-13-rust-1.60.0-nightly".
     let out = Command::new("rustup")
-        .args(&["which", "rustc"])
+        .args(["which", "rustc"])
```

Comment:
This is surprising. What motivated this change?

Reply:
Clippy! I can revert it if you prefer.